scripts/publish-ghpages.sh

```diff
@@ -0,0 +1,27 @@
+#/bin/sh
+
+set -o errexit -o nounset
+
+# User info
+git config user.name "patternfly-build"
+git config user.email "patternfly-build@redhat.com"
+git config --global push.default simple
+
+## Deploy to github pages
+if [ -d "patternfly.github.io" ]; then
+  rm -rf patternfly.github.io
+fi
+git clone git@github.com:patternfly/patternfly.github.io.git
+rsync -av --delete --exclude .git source/_site/ patternfly.github.io
+cd patternfly.github.io
+find components -type f -not -regex ".*/.*\.\(html\|js\|css\|less\)" -print0 | xargs -0 rm
+git add . -A
+if [ -z "$(git status --porcelain)" ]; then
+  echo "Working directory clean, no changes to commit."
+else
+  # Commit generated files
+  git commit -a -m "Added files generated by Travis build"
+  git push origin master:master
+fi
+cd ..
+rm -rf patternfly.github.io
```

Comment:
Same response as above, I want to be able to run this script locally when we need to manually push the site. (eg. travis is unavailable)